hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestChecksumFileSystem.java

```diff
@@ -19,9 +19,8 @@
 package org.apache.hadoop.fs;
 
 import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
 
-import org.apache.hadoop.fs.FileSystem;
-import org.apache.hadoop.fs.FSDataOutputStream;
 import org.apache.hadoop.fs.permission.FsPermission;
 import static org.apache.hadoop.fs.FileSystemTestHelper.*;
```

Comment:
there's a mixup of static and non static imports. This is probably worth fixing while you are cleaning up the unused imports, as it's more unusual

Reply:
I think it is better to leave alone. will cause backporting pain if I try to fix. Will revert back to original.